chi_edge/cli.py

```diff
@@ -159,3 +167,65 @@ def bootstrap(
     return ansible.run(
         "setup_edge_dev.yml", host, group=device_type, host_vars=host_vars
     )
+
+
+@cli.command(help="Bake a balena image")
+@click.argument("image",
+    help=(
+        "Path to the raw disk image to configure. This should be a `.img` "
+        "file."
+    ),
+)
+@click.argument("device_uuid",
+    help=(
+        "The Doni device uuid to configure the image for."
+    ),
+)
+def bake(image, device_uuid):
+    # Ensure we do not overwrite a `config.json` file on the user's system
+    if os.path.isfile("config.json"):
+        print("'config.json' already exists!")
+        return
+
+    # Check for device in doni
+    doni = doni_client()
+    hardware = doni.get(f"/v1/hardware/{device_uuid}/").json()
+    balena_workers = [
+        worker for worker in hardware["workers"]
+        if worker["worker_type"] == "balena"
+    ]
+    if len(balena_workers) != 1:
```

Comment:
I think it's not possible in Doni to have multiple workers of the same type, so it's enough to check for falsy (i.e., 0).